packages/datetime/src/dateInput.tsx

```diff
@@ -100,10 +101,16 @@ export interface IDateInputProps extends IDatePickerBaseProps, IProps {
     /**
      * The position the date popover should appear in relative to the input box.
      * @default Position.BOTTOM
+     * @deprecated since v1.15.0, use `popoverProps.position`
```

Comment:
i don't understand the request here? DRI does not have any deprecated props, nor does NI.